src/player.h

```diff
@@ -764,9 +772,9 @@ class Player final : public Creature, public Cylinder
 			// update all slots
 			std::vector<slots_t> slots = {CONST_SLOT_RIGHT, CONST_SLOT_LEFT, CONST_SLOT_AMMO};
 			for (auto const& slot : slots) {
-				Thing* slotThing = getThing(slot);
+				auto slotThing = getThing(slot);
 				if (slotThing) {
-					Item* slotItem = slotThing->getItem();
+					auto slotItem = slotThing->getItem();
```

Comment:
Could this be `const auto`
```suggestion
					cont auto slotItem = slotThing->getItem();
```